CityOfDetroit/bloom
# fix: release scrub

# Pull Request Template

## Issue Overview

This PR addresses #1603 and #1604


## Description

This PR aligns the error messaging for the User export and Listings export. Additionally, it supplies a temporary solution to the time zone questions by aligning the formatting across all exports and making explicit references to the UTC time zone.

## How Can This Be Tested/Reviewed?

To test all pieces will require running this locally and using the preview links. In order to verify the column name changes and new time formatting, you will have to run this locally. On the other hand, to verify the UTC behavior that is being discussed in the issues, look at the preview link and update a listing. In the export, that updated time will be in UTC.
## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
